beacon_node/client/src/builder.rs

```diff
@@ -208,12 +209,6 @@ where
             builder
         };
 
-        let builder = if let Some(trusted_setup) = config.trusted_setup {
-            builder.trusted_setup(trusted_setup)
-        } else {
-            builder
-        };
```

Comment:
Moved this to the bottom of this method, as config needs to be used so we can't move trusted_setup yet.